.prettierignore

```diff
@@ -5,5 +5,6 @@ generated
 packages/cli/tests/cli/init
 packages/cli/tests/cli/validation
 packages/cli/tests/cli/add
+packages/cli/config/TheGraphNetworksRegistry.json
```

Comment:
Yes, it's a fallback option.

TBH don't think it will ever be needed by anyone since we are relying on being online for bunch of stuff - from querying ABI to deploying the subgraph. And even if there is somebody developing offline with pre-saved ABI, for them even the shipped registry will be enough  in 99% cases since you would only care about network ids and things like RPC and API endpoints are not relevant.

Caching, on the other hand, brings other issues like not having the registry on the first run, complexity with having to deal with cached files, etc

I feel like worrying for a small non-critical use case like that would be over-engineering. We can circle back to it if there is need.

Reply:
Okay I get it :+1: 

It's just that with a flat file, we'll need to consider how often to update it. And that kinda means duplicating some work with PRs to the registry and here. Maybe there's an in-between where the flat file could be stored and handled by the library itself with an interface that allows you to fetch it offline (or throw if not).